src/main/java/org/broadinstitute/consent/http/resources/DatasetResource.java

```diff
@@ -203,15 +209,16 @@ public Response validateDatasetName(@QueryParam("name") String name) {
     public Response getDataSetSample() {
         String msg = "GETting Data Set Sample";
         logger().debug(msg);
-        String fileName = "DataSetSample.tsv";
+        String dataSetSample = "Dataset Name\tData Type\tSpecies\tPhenotype/Indication\t# of participants\tDescription\tdbGAP\tData Depositor\tPrincipal Investigator(PI)\tSample Collection ID\tConsent ID"
+                + "\n(Bucienne Monco) - Muc-1 Kidney Disease\tDNA, whole genome\thuman\tmuc-1, kidney disease\t31\tmuc-1 patients that developed cancer , 5 weeks after treatment\thttp://....\tJohn Doe\tMark Smith\tSC-20658\t1";
```

Comment:
This could be moved to a class level field with getter/setter so it could be customized by a test